### PR TITLE
Remove meta-measured as it is dead upstream

### PR DESCRIPTION
**Pull request by ni-jxie**
### Description

meta-measured layer is no longer being maintained upstream. Migrate the strongswan bbappend from meta-measured into meta-nilrt and remove meta-measured;

[Technical Debt 1564625](https://ni.visualstudio.com/DevCentral/_workitems/edit/1564625)

### Implementation
- Migrate strongswan and trousser to meta-nilrt
- Remove meta-measured from meta-nilrt

### Testing
Pending

### CODE REVIEW

File: recipes-support/strongswan/strongswan_5.%.bbappend [hunk 1]
```diff
@@ -3,7 +3,54 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI += "file://ipsec.init \
 "
```
Comment (shruthi-ravi) [inline recipes-support/strongswan/strongswan_5.%.bbappend:4]: The commit message makes it sound like the strongswan recipe was owned by meta-measured. But it looks like the actual recipe is part of meta-openembedded, and meta-measured only added to it (using a bbappend). Can you modify the message to say that you migrated the strongswan bbappend instead?

File: recipes-support/trousers/trousers_0.3.15.bb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# 0.3.14 unpacks to the current directory instead of to a sub-directory
```
Comment (shruthi-ravi) [inline recipes-support/trousers/trousers_0.3.15.bb]: Is this version-specific comment still needed?

File: recipes-support/trousers/trousers_0.3.15.bb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+S = "${WORKDIR}/trousers-0.3.15"
```
Comment (shruthi-ravi) [inline recipes-support/trousers/trousers_0.3.15.bb]: You can use the recipe variable here. I think they are `PN` and `PV`.

**Review by amstewart — APPROVED** [review]

File: recipes-support/strongswan/strongswan_5.%.bbappend (hunk 1 above)
Comment (ni-jxie) [reply recipes-support/strongswan/strongswan_5.%.bbappend:4]: Done.

**Comment by ni-jxie** [comment]
### PATCH V1

- Remove unneeded version specific comment.
- Update trousers bb to use recipe variable.

File: recipes-support/trousers/trousers_0.3.15.bb (hunk 2 above)
Comment (ni-jxie) [reply recipes-support/trousers/trousers_0.3.15.bb]: Removed

File: recipes-support/trousers/trousers_0.3.15.bb (hunk 3 above)
Comment (ni-jxie) [reply recipes-support/trousers/trousers_0.3.15.bb]: Modified to use `PN` and `PV` instead.

**Review by shruthi-ravi — APPROVED** [review]

**Pull request merged**